lib/documents.js

```diff
@@ -164,6 +185,35 @@ export async function register({
   }
 }
 
+/**
+ * Encrypts the document either once (with the given recipients) or recursively
+ * in a chain (using the recipientChain).
+ *
+ * @param {object} options - Options hashmap.
+ * @param {object} options.document - The document to register.
+ * @param {Array<object>} recipients - See `register()` docstring.
+ * @param {Array<Array<object>>} recipientChain - See `register()` docstring.
+ *
+ * @returns {Promise<object>} Resolves with the JWE, the result of encryption.
+ */
+export async function encrypt({document, recipients, recipientChain}) {
```

Comment:
Why is this exported? If only for testing, let's use an underscore. This should be a private function, right?

Reply:
Sure, changing.